examples/backend_serverpod/example_server/pubspec.yaml

```diff
@@ -8,8 +8,9 @@ environment:
 
 dependencies:
   jaspr: ^0.15.0
-  jaspr_serverpod: ^0.3.2
-  serverpod: 2.0.2
+  jaspr_serverpod:
+    path: ../../../packages/jaspr_serverpod
```

Comment:
No need, melos will link it locally

Reply:
Got it, I’ll go ahead and make the adjustment right away. Thanks for pointing that out :D